Gemfile.lock

```diff
@@ -29,10 +21,9 @@ DEPENDENCIES
   json
   mandate
   minitest (~> 5.11.3)
-  minitest-reporters
   mocha
   pry
   rake
 
 BUNDLED WITH
-   2.0.1
+   2.1.4
```

Comment:
@iHiD Is this okay? Or should we stick with Bundler 2.0.1?

Reply:
It's ok, as we don't use bundler to actually run it in production.

Reply:
Good catch though :)